### ISSUE TITLE
Which lesson to refer to for codealong.

### ISSUE THREAD

**Issue by onyoo** [learn-co-curriculum/layouts-grids-code-along#1]
In the lesson, Jonathan pulls-up the website he has been working on (one file). Throughout this course we have been working on this website via multiple lab files. There is no reference as to which lab's file we should be taking information from. The one Jonathan is using is a more advanced version (ex. there are at least two .php files which we don't have). I believe the correct version to reference is the "adding-responsive-features-v-000" lab, but I am not certain. A note in the video, or on the lesson would make this much clearer.

**Comment by AnnJohn** [comment]
Hi @onyoo sorry for the delay. And yes, that should be the correct version. We'll add a note in this lesson.

**Comment by daniellecroxton** [comment]
Hello, I found the same issue and it doesn't seem that a note has been added to this lesson yet. I had to pause multiple times to make directories in this lab's folder, copy files from the previous codealong and the previous Exceptional Realty project and then import them and my page still isn't looking right. Perhaps if the bootstrap files, videos and images were preloaded in this lab and a note was left to refer to the html and css in adding-responsive-features-v-000 for copying and pasting, it'd be a little easier to focus on updating to Bootstrap rather than chasing down files.
